NtupleProducer/python/runPerformanceNTuple.py

```diff
@@ -245,6 +249,10 @@ def addCalib():
     process.ntuple.objects.L1HFCalo  = cms.VInputTag('l1tPFClustersFromCombinedCaloHF:calibrated')
     process.ntuple.objects.L1HGCalEM = cms.VInputTag('l1tPFClustersFromHGC3DClustersEM', )
 
+def addNNPuppiTaus():
+    process.extraPFStuff.add(process.l1tNNTauProducerPuppi)
+    process.l1pfjetTable.jets.l1tau = cms.InputTag('l1tNNTauProducerPuppi', "L1PFTausNN")
```

Comment:
I suggest just changing the name to `process.l1pfjetTable.jets.nnPuppiTau` since there are other tau collections